src/parser/shared/metrics/apl/index.ts

```diff
@@ -391,12 +391,14 @@ export function aplProcessesEvent(
   event: AnyEvent,
   result: CheckState,
   applicableSpells: Set<number>,
+  playerId: number,
 ): event is BeginChannelEvent | CastEvent {
   return (
     (event.type === EventType.BeginChannel ||
       (event.type === EventType.Cast &&
         event.ability.guid !== result.mostRecentBeginCast?.ability.guid)) &&
-    applicableSpells.has(event.ability.guid)
+    applicableSpells.has(event.ability.guid) &&
+    event.sourceID === playerId
```

Comment:
i guess this is needed because of SEF clones copying your spell casts?

Reply:
Yes otherwise they are all counted as casts